### DISCUSSION TITLE
Увеличить время до эвакуации полицейских машин

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#554] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/billblack/32/763_2.png) [BillBlack](https://forum.octothorp.team/u/BillBlack) ([#4961](https://forum.octothorp.team/p/4961))

Да, говорят, что полиция совсем не РПшит и не играет ранения, вообще ничего не делает. Но все же она играет РП и иногда бывает так, что ты не можешь подойти к машине и прокатиться на ней.

Стоишь ты себе, оказываешь помощь человеку, у него там… ЧМТ, кровотечение, да еще и пневмоторакс вдобавок. Ты взял сумку BLS, оказываешь помощь. Надо везти в больницу, а то скорой нет.
 Бац, машина пропала.

Это конечно хорошо, что вы заботитесь о том, чтобы не кидали машины надолго, но машина полиции - важный артефакт, когда она пропадает, иногда это очень плохо.
 И во время РП ты просто не обращаешь внимания на надпись в чате и не можешь подойти и сесть (да даже это не помогает, ведь ее мало даже чуть двинуть, надо именно ехать) и она пропадает. Это нехорошо.

Предлагаю еще немного увеличить лимит для полицейских машин. Потому что это действительно необходимо офицерам.

- Нужно увеличивать время стоянки;
- Не нужно увеличивать время стоянки.

0голосов

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/mr.nekita/32/296_2.png) [mr.nekita](https://forum.octothorp.team/u/mr.nekita) ([#5243](https://forum.octothorp.team/p/5243))


Всем привет! ![:sunglasses:](https://forum.octothorp.team/images/emoji/twitter/sunglasses.png?v=10 ":sunglasses:")
 Однозначно **ДА**.
 Дополню своими предложениями.

- Машину эвакуирует автоматически через 20 минут, при бездействии. Так же сделать и гражданским авто.
- Поправка, что при запросе эвакуатора он забирает машину через 5 минут, а не 10.
- После смерти авто не пропадает еще 10 минут.
- Лимит увеличить хотя бы для ДПД, или же сержантов. До 7 - 10 крузеров. Убирать его не нужно.


*Я в курсе, что лимиты ввели для предотвращения крашей и низкого фпс, пинга, но это делает машины аналогом автобуса какого-то. Приехал - уехал. Оставить в гараже машину нельзя, нужно бегать каждые 5 минут, чтобы ее не эвакуировали.*

**Reply by msmr-dev** [reply]
че за бред как пользоватся

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/zhilichni/32/641_2.png) [zhilichni](https://forum.octothorp.team/u/zhilichni) ([#7494](https://forum.octothorp.team/p/7494))


Я **за**, будет намного лучше если добавят время на эвакуацию,я думаю никто не будет против.

**Comment by LENFYS** [comment]
За
